extensions/sql-migration/package.json

```diff
@@ -12,6 +12,7 @@
     "azdata": ">=1.19.0"
   },
   "activationEvents": [
+    "*",
```

Comment:
1. If you have a * activation event then you don't need any others
2. Why is this necessary? We have a `onDashboardOpen` event if this was for the new dashboard page you added